network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -117,6 +117,24 @@
                 tw-gap-1
                 tw-grid-flow-row-dense
             ">
+                {% if featured_cta %}
```

Comment:
Same code but relocated from `category_page.html` to this file.

Since both `category_page.html` and `home.html` extends this file, moving the CTA template code here ensures that CTA markup gets generated regardless which page (home or category pages) users land on first.